src/Pages/Templates/TemplatesTable/TemplatesTable.tsx

```diff
@@ -170,6 +171,14 @@ const TemplatesTable = () => {
 
   const countIsZero = count === 0;
 
+  const hasRHELSubscription = subscriptions?.red_hat_enterprise_linux || false;
```

Comment:
If you want a false here, just "double bang"
`!!subscriptions?.red_hat_enterprise_linux`

Reply:
didn't know bang/"double bang" can/should be used like that 😮 resolved [here](https://github.com/content-services/content-sources-frontend/pull/328/commits/92ada247ce4e2cfe19fb4e6adac241f9ae08e54d) 🫡